### ISSUE TITLE
Scroll search UserVars fail to load at startup

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#783]
Lich tries to deserialize the UserVar that contained ScrollInfo, but it doesn't know about the ScrollInfo class. This can delete all of your UserVars.

```
--- Lich: error: undefined class/module Scripting::ScrollInfo
C:/Users/Robert/Desktop/lich/lich.rbw:2308:in `load'
C:/Users/Robert/Desktop/lich/lich.rbw:2308:in `block (2 levels) in <module:Vars>'
C:/Users/Robert/Desktop/lich/lich.rbw:2298:in `synchronize'
```

**Comment by ohbeanz** [comment]
Addressed in PR: https://github.com/rpherbig/dr-scripts/pull/869

**Comment by rpherbig** [comment]
Merged #869
